client/signup/main.jsx

```diff
@@ -104,13 +109,18 @@ class Signup extends React.Component {
 			controllerHasReset: false,
 			login: false,
 			dependencies: props.signupDependencies,
+			siteDomains: props.siteDomains,
+			isPaidPlan: props.isPaidPlan,
 			shouldShowLoadingScreen: false,
 			resumingStep: undefined,
 			loginHandler: null,
 			hasCartItems: false,
 			plans: false,
 			previousFlowName: null,
 		};
+
+		this.isPlanStepFulfilled = false;
+		this.isDomainStepFulfilled = false;
```

Comment:
Are these two values used anywhere? I can't find any usages.

Reply:
You're right, removed them in [c220db7](https://github.com/Automattic/wp-calypso/pull/30242/commits/c220db77ebd6076fe966001d6e23153eb55f5004).